lib/vite-postcss-config.js

```diff
@@ -0,0 +1,41 @@
+const path = require('node:path');
+const fs = require('node:fs');
+const postcssViewportToContainerToggle = require('postcss-viewport-to-container-toggle');
+
+module.exports = async ({
+  assetOptions, util, isPublic = false
+}) => {
+  const postCssConfigPath = path.resolve(process.cwd(), 'postcss.config.js');
+  const importedConfig = isPublic && fs.existsSync(postCssConfigPath)
+    ? await import(postCssConfigPath)
+    : {};
```

Comment:
Only try to get postcss project config file if in public mode.
We might use one in `apostrophe` for other plugins like `autoprefixer`?

Reply:
Autoprefixer for Apos might make sense. I don't think Vite does anything related, I can't recall they are doing it by default (probably inspecting the relevant docs/research a bit is a good idea). If we decide to do that, you can add it directly in the apos base config (`vite-apos-config.js`). It will be merged with anything that potentially comes later.

Reply:
Added autoprefixer only for apos builds as dicsussed